src/nvim/version.c

```diff
@@ -401,7 +401,7 @@ static const int included_patches[] = {
   2083,
   2082,
   2081,
-  // 2080,
+  2080,
```

Comment:
I committed the version.c update from https://github.com/neovim/neovim/pull/36365 to visualize the update from vimpatch.lua.